test/build/resolve-css/snowpack.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  mount: {
+    public: '/',
+  },
+  optimize: {
+    bundle: true,
```

Comment:
This CSS test is two for the price of one: we are testing resolution logic of CSS, which was failing before. But we’re also testing bundling, too, and making sure that `esbuild` is resolving correctly as well.